apiserver/facades/agent/uniter/service.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package uniter
+
+import (
+	"context"
+
+	"github.com/juju/juju/cloud"
+	"github.com/juju/juju/controller"
+	"github.com/juju/juju/core/credential"
+	"github.com/juju/juju/core/model"
+	"github.com/juju/juju/core/network"
+	"github.com/juju/juju/core/watcher"
+	"github.com/juju/juju/environs/config"
+)
+
+// ControllerConfigService is an interface that provides the controller
```

Comment:
**question:** Does the wording `// XXX is an interface that provides YYY` provides different information than the wording `// XXX provides YYY` for an interface ?

I see both of them in this file, which make me feel I miss some point.

**suggestion:** If there is no difference, stick with the later `// XXX provides YYY` the fact that is an interface isn't a thing for primary user of the documentation (they want to know what XXX is meant to do, not how it is implemented.

Reply:
You're right, "is an interface that" is unnecessary, because you can clearly see in the type definition that it is an interface.

I just copied these comments from the concrete service implementations, so these interfaces would be documented instead of nothing. We don't have many "hard rules" around doc comments, they should just be descriptive and tell the developer what something does.